pkgs/applications/misc/plover/default.nix

```diff
@@ -46,6 +46,11 @@ rec {
       sha256 = "0kf27ykw6cjlpyr47nqad5hvpc1sza8x4lpnprwksd70y9hjs40g";
     };
 
+    # upstream recommendation
+    postPatch = ''
+      sed -i 's/^\(\s*plover = plover.\)main:main$/\1dist_main:main/' setup.cfg
```

Comment:
with this change plover still works in a nix-shell
but fails to start in my environment